### PR TITLE
docs: Update package-json.md

### PR DESCRIPTION
**Pull request by p-chan**
Files specified in the bin field are also included. (I execute `npm pack` to confirm.)

### CODE REVIEW

File: docs/lib/content/configuring-npm/package-json.md [hunk 1]
```diff
@@ -285,6 +285,7 @@ Certain files are always included, regardless of settings:
 * `README`
 * `LICENSE` / `LICENCE`
 * The file in the "main" field
+* The file in the "bin" field
```
Comment (ljharb) [inline docs/lib/content/configuring-npm/package-json.md]:
```suggestion
* The file(s) in the "bin" field
```
there can be multiple if it’s an object.

Comment (p-chan) [reply docs/lib/content/configuring-npm/package-json.md]: that's right!

**Comment by wraithgar** [comment]
Why is it two lines now?

**Comment by p-chan** [comment]
> Why is it two lines now?

@wraithgar What do you mean?
I added one line.
Could you please elaborate a bit more?

**Comment by wraithgar** [comment]
Sorry my eyes got a little confused there, I didn't catch that this was `bin` not `main`.

**Review by wraithgar — APPROVED** [review]

**Pull request merged**